src/scss/core/_vars.scss

```diff
@@ -7,3 +7,39 @@ $grid-breakpoints: (
   lg: 992px,
   xl: 1200px
 );
+
+$grid-gutter-width: 15px;
+$grid-gutter-width-lg: 30px;
+
+// colors
+$primary: #00635B;
+$secondary: #FF5A8D;
+$body-color: #151515;
+$light-green: #E0F0E9;
+$light-pink: #F5DADF;
+$gray: #4E4E4E;
+
+// em numbers based of original design base font size 16pixel (mobile and desktop) and converted to em using pxtoem.com
+// font-size
+$h1-font-size: 1.5em;
+$h2-font-size: 1.5em;
+$h3-font-size: 1.125em;
+$h4-font-size: 1em;
+
+// font-size for desktop
+$h1-font-size-lg: 2.25em;
+$h2-font-size-lg: 2em;
+$h3-font-size-lg: 1.5em;
+$h4-font-size-lg: 1.25em;
+
+// line-height
+$h1-line-height: 2.625em;
+$h2-line-height: 2.375em;
+$h3-line-height: 1.75em;
+$h4-line-height: 1.25em;
+
+// line-height for desktop
+$h1-line-height-lg: 1.75em;
+$h2-line-height-lg: 1.75em;
+$h3-line-height-lg: 1.375em;
+$h4-line-height-lg: 1.25em;
```

Comment:
2 errors:

proportions must be taken from the element font size not the the body font size, because line-height em is always referred to the same element font-size value
you are mixing desktop values with mobile ones